metadata.csv

```diff
@@ -239,4 +239,8 @@
 1917	2A	F
 1919	2A	M
 1920	1B	F
+<<<<<<< HEAD
```

Comment:
@Rohan-Kamat this shouldn't be there. You might have missed this while resolving merge conflicts.